### ISSUE TITLE
Support `trim` parameter in density stat

### ISSUE THREAD

**Issue by alshan** [JetBrains/lets-plot#62]

**Comment by PoslavskySV** [comment]
Hi! 

Is it option is still not supported? I'm trying to reproduce this option on [this](http://www.sthda.com/english/wiki/ggplot2-violin-plot-quick-start-guide-r-software-and-data-visualization) example with tooth growth data:

```kotlin
val dose = listOf("0.5", "0.5", "0.5", "0.5", "0.5", "0.5", "0.5", "0.5", "0.5", "0.5", "1.0", "1.0", "1.0", "1.0", "1.0", "1.0", "1.0", "1.0", "1.0", "1.0", "2.0", "2.0", "2.0", "2.0", "2.0", "2.0", "2.0", "2.0", "2.0", "2.0", "0.5", "0.5", "0.5", "0.5", "0.5", "0.5", "0.5", "0.5", "0.5", "0.5", "1.0", "1.0", "1.0", "1.0", "1.0", "1.0", "1.0", "1.0", "1.0", "1.0", "2.0", "2.0", "2.0", "2.0", "2.0", "2.0", "2.0", "2.0", "2.0", "2.0")
val len = listOf(4.2, 11.5, 7.3, 5.8, 6.4, 10.0, 11.2, 11.2, 5.2, 7.0, 16.5, 16.5, 15.2, 17.3, 22.5, 17.3, 13.6, 14.5, 18.8, 15.5, 23.6, 18.5, 33.9, 25.5, 26.4, 32.5, 26.7, 21.5, 23.3, 29.5, 15.2, 21.5, 17.6, 9.7, 14.5, 10.0, 8.2, 9.4, 16.5, 9.7, 19.7, 23.3, 23.6, 26.4, 20.0, 25.2, 25.8, 21.2, 14.5, 27.3, 25.5, 26.4, 22.4, 24.5, 24.8, 30.9, 26.4, 27.3, 29.4, 23.0)

letsPlot(mapOf("dose" to dose, "len" to len)) {
    y = "len"
    fill = "dose"
} + geomViolin(trim=false)
```

and still obtain trimmed plots:

<img width="618" alt="image" src="https://user-images.githubusercontent.com/3834261/171854788-b5d349fa-f5bd-49fc-95c6-4926e6b26281.png">

**Comment by alshan** [comment]
We'll take a look, thanx!

**Comment by ASmirnov-HORIS** [comment]
Fixed in [904ac24](https://github.com/JetBrains/lets-plot/commit/904ac241c34b8a06113612d171654168c2f3792c) and [f420324](https://github.com/JetBrains/lets-plot/commit/f42032402acf94ede91a7e97caedf1f1cce869ca).
